apps/www/src/state/plugin/viewCollection.js

```diff
@@ -1,28 +1,77 @@
-import { hydra, schema } from '@tpluscode/rdf-ns-builders'
-import { newReference } from '../../clownface.js'
+import { hydra, rdf, schema } from '@tpluscode/rdf-ns-builders'
+import * as ns from '../../ns.js'
 
 export const viewCollection = {
   model: {
+    state: {},
+    reducers: {
+      setPointer(state, pointer) {
+        return { ...state, pointer }
+      },
+      setNewViewOperation(state, newViewOperation) {
+        return { ...state, newViewOperation }
+      },
+      setNewViewShape(state, newViewShape) {
+        return { ...state, newViewShape }
+      },
+    },
     effects(store) {
       const dispatch = store.getDispatch()
 
       return {
-        async deleteView(pointer) {
-          const { resource, core } = store.getState()
+        'core/setContentResource': ({ pointer }) => {
+          if (pointer.has(rdf.type, ns.viewBuilder.ViewCollection).term) {
+            dispatch.viewCollection.setPointer(pointer)
+            dispatch.viewCollection.loadNewViewShape()
+          }
+        },
+        async loadNewViewShape() {
+          const {
+            core,
+            resource,
+            viewCollection: { newViewShape },
+          } = store.getState()
+
+          if (!newViewShape) {
+            const collection = resource.representations.get(core.contentResource.id).root
+            const operation = collection.findOperations({
+              bySupportedOperation: schema.CreateAction,
+            }).shift()
+
+            const promise = operation.expects.shift().load()
+
+            dispatch.viewCollection.setNewViewOperation(operation)
+            dispatch.viewCollection.setNewViewShape(promise)
+
+            promise.then(({ representation }) => {
+              dispatch.viewCollection.setNewViewShape(representation.root)
+            })
+          }
+        },
+        'operation/succeeded': ({ operation, response }) => {
+          const {
+            viewCollection: { newViewOperation },
+          } = store.getState()
+
+          if (operation === newViewOperation && response.createdResourceUri) {
+            const created = response.createdResourceUri
+            dispatch.routing.goTo(created)
+          }
```

Comment:
When the operation succeeds, navigate to the view created (this would be HTTP header `Location`)